### PR TITLE
vlatex: added brackets in multiplication

### PR DESCRIPTION
**Pull request by maurogaravello**
According to issue 12078, vlatex did not work correctly
in the case of multiplication, where one term is a derivative
of a sum.
The changes are in the files
modified:   sympy/physics/vector/printing.py
modified:   sympy/physics/vector/tests/test_printing.py

The change is in _print_Derivative in printing.py
A test is added in test_printing.py

<!-- If this pull request fixes an issue please indicate which issue by typing "Fixes #NNNN" below.>

### CODE REVIEW

File: github_deploy_key.enc [hunk 1]
```diff
@@ -1 +0,0 @@
...XFf3fHbNRKYGWgr3t0dOa5yiazBD7wV7QUVlPzr8hpkpXPREVJyZWiN6z7Txcd3ZhPlRtW6G2VtEJSVeVfTetlGX347ZUCtWBVmO2U6FqYg3de53vzslicug8tR3_Wrlsbnuuz27VS4R-5t1Ckwaf6EP2q5hyOc4AzWcTPxtm1LRumO_b47i36kwDYzaG5toYiloR0IE_4P9dpnEbUPc971r1V6H5suG1I3dICo74afik8jN7_AU9FrQdqzaXPa-BEh252bNhelWZZmEGIVzwqwAUbvbb1pRYWa9BhmsYivxgl5XU5_UXpM8s6t6IRy2D9PTr0Al4Emj_7W07Wq6iWHHkvBjT2Eye4SfOC9_viJmTeps3XPYXYyQKTctbdRc4w5zQL6zvtfVrG6jcR9OZt2b1lHKYStxQaVtJmauiVYwlkA9Q52DcYbIETGJGBeMClZMHBpiStURLyI6mtHPpYttA0pZR2jwjc1co9szhPPGYO3jHv0PjhGz8rgUaHYn9dYgmgfyYpsIRfJZz2IZhz6i9ir6dp-KlObMAbmXGkgtw2yXx95Aj1fOwtmSWXXkbjCH6WSk4evBcUR7NZpM_8tFfbbELkONva5dRSc_Q_Z3NFyPtW68Dz3TmTzDyFX1-B6ysHxuG3OlnE1f1ZAPpKpeoiZ4uMmNQ2Geuj8W7sDazKi88sR-Oj_ynk319XyzQbyoRFeEIII5KQANnLPSbxRQIvD_87mYH8QVKngjTAsNTv-LCMPOaLgqcv6dB9ATIp9vxz6JfEJwORRZ1XqTpOWsj9OslTq56AnsmwQTxAQm2lAYiCSaRWR9x9wSPaz6O6Qa-VYmoEjuvaOlHN32VSRAwK0t2Wcm2a_O7PT0N4tg9s-z1S48fhmGMePqfZZrYX3x-dr6mKd8WDwKf1YY6yAv6Tk4H1hhyEwPeZPq36_5gzw0haY0PK9xuKAAn4MKCY468zr-9kuI7l-nFVQHz3O62aa_CDNYh5DEbnd0q7Oi3LTfAdcsYNnpXc2AbrAkSCI9fOV1DoRiR-Tr77zERvs5J2xIo_k0TSzaBrA4zpslVMo1BLxKbvTQ7bjw8UEsEKln1UVP8dDbjckZ4NN0RZMM3qv2hpN2D4tx_SghjjpRsqRQ7TZEOJMEgVrKgo3hT1ZDeBJSHJRqriF_XyphshcqdGRGU82ADKHuPdjkL6cuS1M=
```
Comment (gxyd) [inline github_deploy_key.enc]: There would be no need to modify this file.

**Comment by gxyd** [comment]
Apart from one comment, this PR looks okay.

File: github_deploy_key.enc (hunk 1 above)
Comment (maurogaravello) [reply github_deploy_key.enc]: Also for me is ok. 
Should I do anything else?

**Comment by maurogaravello** [comment]
Also for me is ok.
Should I do anything else?

**Comment by maurogaravello** [comment]
Now should be ok.

**Comment by gxyd** [comment]
Looks good to me. Merging, thanks.

**Pull request merged**